src/app/plan/components/Wizard/GeneralForm.tsx

```diff
@@ -24,10 +24,15 @@ const GeneralForm: React.FunctionComponent<IGeneralFormProps> = ({
   touched,
   values,
 }: IGeneralFormProps) => {
+  const inputRef = useRef(null);
   const onHandleChange = (val, e) => {
     handleChange(e);
   };
 
+  useEffect(() => {
```

Comment:
As written, this will focus the input field every time this component renders, instead of just the first time. If we ever add any other fields here that could become problematic. If you want it to focus only after the first render you can pass an empty dependencies array as the second argument of `useEffect`. https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect